src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -102,5 +102,12 @@ private void createDir() {
             directory.mkdir();
         }
     }
+
```

Comment:
I just realised `MESSAGE_USAGE` of `ExportCommand` contains 'Invalid command format!\n' message, it should just contain the usage message similar to the `MESSAGE_USAGE` from other `XYZCommand`, so it will follow the current AB3 convention.
Can refer to the `AddCommand`.